src/nvim/ex_docmd.c

```diff
@@ -1480,6 +1480,31 @@ bool is_cmd_ni(cmdidx_T cmdidx)
                                      || cmdnames[cmdidx].cmd_func == ex_script_ni);
 }
 
+// Find the command name after skipping range specifiers.
+static char *find_command_after_range(exarg_T *eap)
+{
+  char *cmd = eap->cmd;
```

Comment:
do not drop useful information. this comment from the old code was a useful hint because it's not obvious that this has anything to do with "modifiers"
 
```suggestion
  // Save location after command modifiers.
  char *cmd = eap->cmd;
```